{"name": "test_LongParam", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <test_plugins.Test_plugins object at 0x00000229948DDF70>\n\n    def test_LongParam(self):\n        name1 = \"LongParam\"\n        self.p2 = modify_conf_file()\n        self.p2.SetPluginParams()\n        self.p1.send_cmd(\"tftp -g -r testserver.conf 192.168.1.65\")\n        rel=self.p1.send_cmd(\"kill -USR1 `pidof testserver`\")\n        rel=self.p1.send_enter()\n        print(rel)\n        self.p1.log(rel,name1,self.nowtime)\n        time.sleep(1)\n        rel = re.findall(r'\"Result\":(\\S)',rel)\n        print(rel)\n        print(type(rel))\n>       if rel[0] == \"0\":\nE       IndexError: list index out of range\n\ntest_plugins.py:35: IndexError"}, "attachments": [{"name": "log", "source": "dcf4dbe1-7e15-4652-92a4-f4000654e9f3-attachment.txt", "type": "text/plain"}], "start": 1700730617320, "stop": 1700730618447, "uuid": "f409b5aa-c1f6-4ae3-9df7-759b18bda2eb", "historyId": "ada529718f23cb5a4760f3e163b5afbe", "testCaseId": "ada529718f23cb5a4760f3e163b5afbe", "fullName": "test_plugins.Test_plugins#test_LongParam", "labels": [{"name": "suite", "value": "test_plugins"}, {"name": "subSuite", "value": "Test_plugins"}, {"name": "host", "value": "l29707B"}, {"name": "thread", "value": "23956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_plugins"}]}